### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by benjmyers**

### CODE REVIEW

File: src/exercise/04.html [hunk 1]
```diff
@@ -6,17 +6,31 @@
   <script src="https://unpkg.com/react@17.0.0/umd/react.development.js"></script>
   <script src="https://unpkg.com/react-dom@17.0.0/umd/react-dom.development.js"></script>
   <script src="https://unpkg.com/@babel/standalone@7.12.4/babel.js"></script>
+  <script src="https://unpkg.com/prop-types@15.7.2/prop-types.js"></script>
+
   <script type="text/babel">
     // 🐨 Make a function called `message` which returns the JSX we want to share
+    const Message = ({subject, greeting}) => {
```
Comment (goodguyry) [inline src/exercise/04.html:13]: We can also use an implicit return here.

```javascript
const Message = ({ subject, greeting }) => <div className="message">{greeting}, {subject}</div>
```

File: src/exercise/04.html [hunk 2]
```diff
@@ -6,17 +6,31 @@
...
     // 🐨 use that function in place of the divs below with:
     // 💰 {message({children: 'Hello World'})} {message({children: 'Goodbye World'})}
     const element = (
-      <div className="container">
-        <div className="message">Hello World</div>
-        <div className="message">Goodbye World</div>
-      </div>
-    )
+      <React.Fragment>
+        <Message greeting={3} />
```
Comment (goodguyry) [inline src/exercise/04.html]: Looks like this one got borked along the way 🙃

**Review by goodguyry — APPROVED** [review]
🍎

File: src/exercise/04.html [hunk 3]
```diff
@@ -6,17 +6,31 @@
...
 
     // 🐨 use that function in place of the divs below with:
     // 💰 {message({children: 'Hello World'})} {message({children: 'Goodbye World'})}
     const element = (
-      <div className="container">
-        <div className="message">Hello World</div>
-        <div className="message">Goodbye World</div>
-      </div>
-    )
+      <React.Fragment>
```
Comment (kaitlinbolling) [inline src/exercise/04.html]: A [short syntax](https://reactjs.org/docs/fragments.html#short-syntax) you'll sometimes see for `<React.Fragment>` is `<>` and `</>`. For example:

```
<>
  <Message greeting="Hello" subject="World" />
  <Message greeting="Goodbye" subject="World" />
</>
```

**Pull request merged**